configure

```diff
@@ -583,6 +618,12 @@ while [[ ! -z $1 ]] ; do
 #      HDF5LIB="-I$VALUE/include $VALUE/lib/libhdf5_hl.a $VALUE/lib/libhdf5.a -lm"
 #      echo "Using HDF5LIB: $HDF5LIB"
 #      ;;
+    "--with-readline")
```

Comment:
> but readline here is slightly different in that a copy is embedded inside of the cpptraj source tree, unlike those others.

The older copy of readline in the git tree is in `AmberTools/src/gleap/freelib/src`.  Since that isn't distributed anymore, it was copied to the cpptraj source tree.